modyn/supervisor/internal/triggers/cost/costtrigger.py

```diff
@@ -29,13 +29,10 @@ def __init__(self, config: CostTriggerConfig):
         self.config = config
```

Comment:
unrelated to this PR but currently we have both a cost_trigger and costtrigger.py living in this directory

Reply:
also the decision policy.py file seems to not contain any code

Reply:
I wasn't fully sure whether to put the cost triggers (3x) into the /triggers directory or into a separate directory. For the other triggers, the trigger-specific subdirectories have only contained additional logic so far, and we have only had one `*trigger.py` for every policy type. When refining the docs PR earlier, I decided to put those files into `/triggers`. Thanks for the reminder here :)